src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -3428,7 +3435,7 @@ bool DoorLockServer::HandleRemoteLockOperation(chip::app::CommandHandler * comma
     }
 
     // credentials check succeeded, try to lock/unlock door
-    success = opHandler(endpoint, pinCode, reason);
+    success = opHandler(commandObj, endpoint, pinCode, reason);
```

Comment:
What is the scope of `commandObj`? Here a pointer to commandObj is being passed (and subsequently is passed to the application). BUT, if the application defers the callback to setLockState(...commandObj...) and said callback is made asynchronously later in time, will the pointer to commandObj still be valid? If not, I would suggest perhaps passing the FabricIndex and NodeId values themselves (since this is all that is being used from commandObj anyway).

Reply:
Changed it to pass `fabricIdx` and `nodeId` directly.